Source/WebCore/page/scrolling/ScrollAnchoringController.cpp

```diff
@@ -50,35 +53,138 @@ ScrollOffset ScrollAnchoringController::computeOffset(RenderBox& candidate)
     return ScrollOffset(FloatPoint(candidate.absoluteBoundingBoxRect().location() - m_frameView.layoutViewportRect().location()));
 }
 
-void ScrollAnchoringController::chooseAnchorElement()
+static Element* anchorElementForPriorityCandidate(Element* element)
 {
-    // TODO: fully implement anchor node selection algorithm in spec
-    // (specifically excluded subtrees and priority nodes)
-    // https://drafts.csswg.org/css-scroll-anchoring/#anchor-node-selection
+    while (element) {
+        if (auto renderer = element->renderBox()) {
```

Comment:
Is there a reason to use `renderBox()` and not `renderer()`?

Reply:
Right should be `renderer()`.